### PR TITLE
GHP Schema Edits

### PR DESCRIPTION
**Pull request by tanushree04**
- Adds more descriptions/edits for URBANopt

### CODE REVIEW

**Comment by tanushree04** [comment]
@atpham88 , I will update this when we have capital cost information for BAU case.

File: schemas/input_schema/REopt-BAU-input.json [hunk 1]
```diff
@@ -53,9 +61,10 @@
         }
     },
     "ExistingChiller": {
-        "type": "float",
-        "required": false,
-        "description": "Only required if BAU system uses fuel for cooling. No need to define anything additional here"        
+        "fuel_cost_per_mmbtu": {
```
Comment (atpham88) [inline schemas/input_schema/REopt-BAU-input.json]: There's no "fuel_cost_per_mmbtu": {} needed for ExistingChiller. If a building uses fuel for heating, it is assumed to use fuel for heating as well and the fuel cost is already in ExistingBoiler

**Review by atpham88 — CHANGES_REQUESTED** [review]

File: schemas/input_schema/REopt-BAU-input.json (hunk 1 above)
Comment (tanushree04) [reply schemas/input_schema/REopt-BAU-input.json]: thanks, I removed the existing chiller in the schema

File: schemas/input_schema/REopt-BAU-input.json [hunk 2]
```diff
@@ -59,12 +59,5 @@
             "required": false,
             "description": "Only required if BAU system use fuel for heating. In this case, fuel cost needs to be specified"
         }
-    },
-    "ExistingChiller": {
```
Comment (tanushree04) [inline schemas/input_schema/REopt-BAU-input.json:55]: @atpham88 removed this field

**Review by atpham88 — APPROVED** [review]

**Pull request merged**